htdocs/compta/bank/bankentries_list.php

```diff
@@ -1209,10 +1210,11 @@
 					print ' '.$paymentscstatic->getNomUrl(2);
 				} elseif ($links[$key]['type'] == 'payment_vat')
 				{
-					$paymentvatstatic->id = $links[$key]['url_id'];
-					$paymentvatstatic->ref = $links[$key]['url_id'];
-					print ' '.$paymentvatstatic->getNomUrl(2);
-				} elseif ($links[$key]['type'] == 'payment_salary')
+                    print '<a href="'.DOL_URL_ROOT.'/compta/payment_vat/card.php?id='.$links[$key]['url_id'].'">';
```

Comment:
Why removing a link with getNomUrl (recommended) with a hard coded value (must be avoid) ?

Reply:
@eldy i did new feature on 12 dolibarr version, it was a merge mistake, i just corrected that, is it ok for you ?